src/indexes/customIndexes.js

```diff
@@ -143,6 +143,66 @@ module.exports = {
                 }
             }
         ],
+        'Account_4_0_0': [
+            {
+                keys: {
+                    'subject._uuid': 1,
+                    '_uuid': 1
+                },
+                options: {
+                    name: 'account.subjectUuid_uuid'
+                }
+            },
+            {
+                keys: {
+                    'subject._sourceId': 1,
+                    '_uuid': 1
+                },
+                options: {
+                    name: 'account.subjectSourceid_uuid'
+                }
+            }
+        ],
+        'AllergyIntolerance_4_0_0': [
+            {
+                keys: {
+                    'patient._uuid': 1,
+                    '_uuid': 1
+                },
+                options: {
+                    name: 'allergyIntolerance.patientUuid_uuid'
+                }
+            },
+            {
+                keys: {
+                    'patient._sourceId': 1,
+                    '_uuid': 1
```

Comment:
We can combine the indexes for patient and subject since these 2 fields are very frequent across most of the Patient linked resources.